django_ipam/tests/base/test_models.py

```diff
@@ -169,3 +169,27 @@ def test_retrieves_ipv6_ipnetwork_type(self):
         instance = self._create_subnet(subnet='2001:db8::0/32')
         instance = self.subnet_model.objects.get(pk=instance.pk)
         self.assertIsInstance(instance.subnet, IPv6Network)
+
+    def test_incompatible_ipadresses(self):
+        failed = True
+        instance = self._create_subnet(subnet='10.1.2.0/24')
+        try:
+            self._create_subnet(subnet='2001:db8::0/32', master_subnet=instance)
+        except TypeError as err:
+            self.assertEqual(str(err), '2001:db8::/32 and 10.1.2.0/24 are not of the same version')
+            failed = False
+        if failed:
+            self.fail('TypeError not raised')
+
+    def test_ipadresses_missing_attribute(self):
+        failed = True
+        instance = self._create_subnet(subnet='10.1.2.0/24')
+        instance2 = self._create_subnet(subnet='10.1.3.0/25')
+        try:
+            del instance.subnet.network_address
+            instance.subnet.subnet_of(instance.subnet)
+        except AttributeError as err:
+            self.assertIn(str(err), '\'IPv4Network\' object has no attribute \'network_address\'')
```

Comment:
This is unexpected error message because it looks like a small overlook in the library's code.
Attribute error is raised only when `network_address` is deleted. (`version` and `broadcast_address` are protected and can't be deleted by user)
But when `network_address` is deleted, TypeError can't access the `a` and `b` values.
That raises another attribute error which i have catched and printed here.

Reply:
ok